pkg/internal/checkup/executor/console/console.go

```diff
@@ -36,6 +36,13 @@ type vmiSerialConsoleClient interface {
 	VMISerialConsole(namespace, name string, timeout time.Duration) (kubecli.StreamInterface, error)
 }
 
+type Expecter struct {
```

Comment:
Could you please add a few words in the commit message about this new struct?

Reply:
DONE